_posts/2018-12-19-NUnit3TestAdapter-3.12.md

```diff
@@ -1,6 +1,6 @@
-NUnit3TestAdapter version 3.11.2 is released.
+NUnit3TestAdapter version 3.12.0 is released.
 
-This release focus on two enhancements, generating NUnit3 Test Results xml and embedding the engine to avoid clashes with other packages.
+This release focuses on two enhancements: generating NUnit3 Test Results XML and updating the NUnit engine which avoids clashes with other packages.
```

Comment:
We didn't embed the engine yet. That would be https://github.com/nunit/nunit3-vs-adapter/issues/583.